ros2service/ros2service/verb/list.py

```diff
@@ -22,7 +22,7 @@ class ListVerb(VerbExtension):
     """Output a list of available services."""
 
     def add_arguments(self, parser, cli_name):
-        add_arguments(parser)
+        add_strategy_node_arguments(parser)
```

Comment:
This change doesn't seem to be related to the topic of the PR. Please consider separating it.